setupext.py

```diff
@@ -1261,11 +1261,13 @@ def get_extension(self):
         sources = [
             'src/ft2font.cpp',
             'src/ft2font_wrapper.cpp',
-            'src/mplutils.cpp'
+            'src/mplutils.cpp',
+            'src/py_converters.cpp',
             ]
         ext = make_extension('matplotlib.ft2font', sources)
         FreeType().add_flags(ext)
         Numpy().add_flags(ext)
+        LibAgg().add_flags(ext)
```

Comment:
py_converters has Agg stuff in it so includes its headers, and then we need to add its flags... I think I can split them up so that this isn't necessary.

Using `p` will still produce an `int` which caused the warnings on AppVeyor.

Reply:
Though maybe I'd wait for #10426 so I don't have to worry about the old backends.

Reply:
but we can just do the cast of the result of `p` ourselves?
I don't want to force you to overengineer this, the addition of agg as a dependency only somewhat annoys me.

Reply:
But then we'd need all the `!= 0` that you didn't like, no? Because [MSVC didn't like `(bool)var`](https://ci.appveyor.com/project/matplotlib/matplotlib/build/1.0.6406/job/l1613vk1guu0hlf8#L1026) either...

Reply:
if an explicit typecast triggers a warning that seems a bit too much but can't do anything about it :/ I guess that sort of forces towards using `!= 0` (or `!!x`) then...

Reply:
If it helps you feel a little better, I also stopped it adding Agg sources to these extensions, which was unnecessary.